atlascli.md

```diff
@@ -1,13 +1,11 @@
-# Atlas CLI (Pre-Release)
+# Atlas CLI
 
 The MongoDB Atlas CLI is a modern command line interface that enables you to manage MongoDB Atlas from the terminal.
 
 ![atlascli-atlas-quickstart](https://user-images.githubusercontent.com/5663078/156184669-57c8ddce-6f0a-4e84-9311-2d996cb27942.gif)
 
 ## Installing
 
-Atlas CLI is currently in the pre-release phase. Do not use it in a production environment.
```

Comment:
Should we still say something like - this is the first release of AtlasCLI, use with care? or please report any bugs

Reply:
v1.0.0 is a stable release so customers should be able to use it without any special precautions